shpc/main/modules/template.py

```diff
@@ -46,7 +46,7 @@ def substitute(self, template):
         For all known identifiers, substitute user specified format strings.
         """
         subs = {
-            "{|module_name|}": self.settings.module_name or "{{ parsed_name.tool }}"
+            "{|module_name|}": self.settings.module_name or "{{ config.name.tool }}"
```

Comment:
Oops I think I should have changed this back! Technically they would both work, but since we are providing parsed_name for the user (from settings.yaml -> template) it's good to be consistent.

```suggestion
            "{|module_name|}": self.settings.module_name or "{{ parsed_name.tool }}"
```